run.py

```diff
@@ -177,7 +182,9 @@ def _validate_devices(devices: str):
     parser.add_argument("--cudastreams", action="store_true",
                         help="Utilization test using increasing number of cuda streams.")
     parser.add_argument("--bs", type=int, help="Specify batch size to the test.")
-    parser.add_argument("--flops", choices=["model", "dcgm"], help="Return the flops result")
+    parser.add_argument("--flops", choices=["model", "dcgm"], help="Return the flops result.")
+    parser.add_argument("--export-all-records", type=str, action="store",
```

Comment:
let's remove the `type=str` part and use `action='store_true'`

Reply:
solved by https://github.com/pytorch/benchmark/pull/947/commits/af83645cb679ca3b18ad318c86346a2f882ebe77